lib/model.js

```diff
@@ -2740,11 +2726,9 @@ Model.findByIdAndUpdate = function(id, update, options, callback) {
  * If you need full-fledged validation, use the traditional approach of first
  * retrieving the document.
  *
- *     Model.findById(id, function (err, doc) {
- *       if (err) ..
- *       doc.name = 'jason bourne';
- *       doc.save(callback);
- *     });
+ *     const doc = await Model.findById(id)
+ *     doc.name = 'jason bourne';
+ *     await doc.save(callback);
```

Comment:
We should choose between `await` or `.save(callback)`.

Reply:
forgot to change this

Reply:
fixed in 099db0094c756941c68c68d80643d454915a5653

Reply:
Thanks 👍